sdks/python/apache_beam/examples/cookbook/bigtableio_it_test.py

```diff
@@ -142,6 +144,9 @@ def setUp(self):
       self.instance.create(clusters=[cluster])
     self.table = self.instance.table(self.table_id)
 
+    # We need a small wait for the BigTable instance to start and stabilize.
```

Comment:
Is there a way to ping the instance instead? This would make it faster if it doesn't take 5 seconds, and also would be robust to  long tail latencies.